examples/bridge-app/esp32/README.md

```diff
@@ -151,6 +151,20 @@ make sure the IDF_PATH has been exported(See the manual setup steps above).
 
           $ idf.py -p /dev/tty.SLAB_USBtoUART monitor
 
+## Setting up chip-tool
```

Comment:
Please make similar changes in other examples as well

Reply:
done